R/check.R

```diff
@@ -69,15 +76,16 @@ check_group_date_unique <- function(obs) {
 #' @return NULL
 #'
 #' @family check
+#' @importFrom rlang abort
 check_module <- function(module) {
   if (!"data" %in% names(module)) {
-    stop(
+    rlang::abort(
       "Must contain a list component specifying the data requirements for
        further modelling as a list"
     )
   }
   if (!is.list(module[["data"]])) {
-    stop(
+    rlang::abort(
```

Comment:
My opinion: while in here, let's bring these up to style guide standards and say *what* must contain etc (e.g "Argument `module` ...") and similarly create a complete sentence, code element identified, etc for the message below.

*Not* a call for full review of these, just minor see-something, fix-something as we're doing all the error messaging.

Also need to update the STYLE_GUIDE.md to reflect usage of rlang.

Reply:
Per overall comment: perhaps warrants own PR. @seabbs - thoughts? if so, let's try to capture notes / suggestions I made over this PR in an issue / update to STYLE_GUIDE.

Reply:
I think we should capture notes here and make suggestions in a new issue vs addressing here.

Reply:
> ) and similarly create a complete sentence, code element identified, etc for the message below.

Can we make a new issue for this. 

> Also need to update the STYLE_GUIDE.md to reflect usage of rlang.|

I think this needs to be in this PR.